app/services/framework_responses/bulk_updater.rb

```diff
@@ -48,7 +53,7 @@ def build_value_updates_list
 
     def apply_bulk_response_changes(updated_responses)
       # Bulk update all modified response values
-      FrameworkResponse.import(updated_responses, validate: false, on_duplicate_key_update: { conflict_target: [:id], columns: %i[value_text value_json responded] })
+      FrameworkResponse.import(updated_responses, validate: false, on_duplicate_key_update: { conflict_target: [:id], columns: %i[value_text value_json responded responded_by responded_at] })
```

Comment:
Just double checking - `section` is not included here because the section does not change after the framework question gets created?

Reply:
yes, section is only added on creation of responses, and will never change